independent-projects/tools/message-writer/src/main/java/io/quarkus/devtools/DefaultMessageWriter.java

```diff
@@ -0,0 +1,57 @@
+package io.quarkus.devtools;
```

Comment:
If `message-writer` belongs to a different module, it must have its own namespace

Reply:
ok on it :)

Reply:
done